SnaffCore/ActiveDirectory/AdData.cs

```diff
@@ -16,11 +16,11 @@ public class AdData
         private List<string> _domainUsers = new List<string>();
         private List<DFSShare> _dfsShares = new List<DFSShare>();
         private List<string> _dfsNamespacePaths;
-        private Domain _currentDomain;
-        private string _domainName;
-        private string _targetDomain;
+//        private Domain _currentDomain;
+//        private string _domainName;
+        private Domain _targetDomain;
         private string _targetDc;
-        private DirectorySearch _directorySearch;
+        private string _targetDomainNetBIOSName;
```

Comment:
You seem to have replaced all of the "raw LDAP" comms stuff that I "borrowed" from SharpHound with the "System.DirectoryServices.ActiveDirectory" stuff that causes problems when trying to run via a SOCKS proxy. Was this possibly a holdover from the code you wrote before the last big update?

Reply:
Ah,  I didn't realize that was why you were avoiding the Domain class.  I should be able to move that back.

Reply:
I'm not sure I'm following.  I changed _targetDomain to a Domain class instead of a string so that I could take some other properties from it (e.g., PDCEmulator),  but the retrieval is still using your original:

```
_targetDomain = Domain.GetCurrentDomain();
```